setup.py

```diff
@@ -59,4 +59,4 @@ def is_exe(fpath):
     install_requires=install_requires,
     include_package_data=True,
     zip_safe=False,
-)
+)
```

Comment:
I think this is a mistake here, the line at the end of file shouldn't be removed.

Reply:
I will check that.